src/main/resources/application.yaml

```diff
@@ -38,6 +42,23 @@ spring:
 #          lob:
 #            # silence the 'wall-of-text' - unnecessary exception throw about blob types
 #            non_contextual_creation: true
+  security:
+    oauth2:
+      client:
+        provider:
+          oidc:
+            issuer-uri: ${IDAM_OIDC_URL:http://localhost:5000}/o
+        registration:
+          oidc:
+            client-id: internal
+            client-secret: internal
+idam:
+  api:
+    url: ${IDAM_API_URL:http://localhost:5000}
+  s2s-authorised:
+    services:
```

Comment:
We can probably add `xui_webapp` as a authorised service as a starter for 10. i.e. based on the [CPO confluence doc](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1478700538#CasePaymentOrders-AccessControl)  

And if adding using an environment variable (e.g. `CPO_S2S_AUTHORISED_SERVICES`) then this may also need adding to `charts/cpo-case-payment-orders-api/values.yaml` if MCA is a good guide.